### ISSUE TITLE
New: rule prefer-replace-text

### ISSUE THREAD

**Issue by Aladdin-ADD** [not-an-aardvark/eslint-plugin-eslint-plugin#47]
this rule will warn about:
```js
context.report({
  fix(fixer) {
    return fixer.replaceTextRange(node.range, "");
  }
});

context.report({
  fix(fixer) {
    return fixer.replaceTextRange([node.range[0], node.range[1]], "");
  }
});
```

**Comment by not-an-aardvark** [comment]
Nitpick: I think `prefer-replace-text` would be a better name for consistent capitalization (even though the method is actually called `replaceText`.

**Comment by Aladdin-ADD** [comment]
I'm a little worried it will cause some false positive cases. e.g
```js
  fix(fixer) {
    node.range= [start, end];
    return fixer.replaceTextRange(node.range, "");
  }
```

**Comment by not-an-aardvark** [comment]
I think that would be fine because `replaceTextRange` just uses the `range` property anyway.
